qa/acceptance-tests/src/test/java/io/camunda/it/auth/RoleAuthorizationIT.java

```diff
@@ -536,5 +624,24 @@ private static MappingSearchQueryResult searchMappingRuleByRole(
     return OBJECT_MAPPER.readValue(response.body(), MappingSearchQueryResult.class);
   }
 
+  // TODO: will be removed in the next PR for client search
```

Comment:
👍 Could you make sure you also remove the dependency you've added in the pom file?